app/controllers/annotations_controller.rb

```diff
@@ -75,6 +75,8 @@ def annotation_params
     params[:annotation].delete(:id)
     params[:annotation].delete(:created_at)
     params[:annotation].delete(:updated_at)
+    # Prevent spoofing the submitter
+    params[:annotation][:submitted_by] = @current_user.email
```

Comment:
If we are overwriting `submitted_by` here, then we should be able to get rid of `annObj.submitted_by` in `annotations.js` https://github.com/autolab/Autolab/blob/14bc1f47c8f5c8682985ae18059cb80b68147ac2/app/assets/javascripts/annotations.js#L513

Reply:
Yes, good catch! I've removed that too.

Reply:
One last thing, we can get rid of `submitted_by` here too:

https://github.com/autolab/Autolab/blob/717a9153c2debb5d9b12b78547bf50f95655af04/app/controllers/annotations_controller.rb#L78

Reply:
Nevermind, it's needed since we are setting `:submitted_by` right before this line